qt/search_panel.cpp

```diff
@@ -195,11 +199,24 @@ bool SearchPanel::TryTrafficSimplifiedColorsCmd(QString const & str)
   return true;
 }
 
-void SearchPanel::OnSearchTextChanged(QString const & normalized)
+std::string SearchPanel::GetCurrentInputLocale()
+{
+  switch (QGuiApplication::inputMethod()->locale().language())
+  {
+  case QLocale::Language::Belarusian: return "be";
+  case QLocale::Language::German: return "de";
+  case QLocale::Language::Russian: return "ru";
+  case QLocale::Language::Ukrainian: return "uk";
+  default: return "en";
+  }
+}
```

Comment:
```suggestion
  return QGuiApplication::inputMethod()->locale().name().first(2);
```

[QString](https://doc.qt.io/qt-6/qstring.html) QLocale::name() const

Returns the language and country of this locale as a string of the form "language_country", where language is a lowercase, two-letter ISO 639 language code, and country is an uppercase, two- or three-letter ISO 3166 country code.